pretyping/cbv.ml

```diff
@@ -455,7 +455,8 @@ let cbv_norm infos constr =
 (* constant bodies are normalized at the first expansion *)
 let create_cbv_infos flgs env sigma =
   let infos = create
-    (fun old_info tab c -> cbv_stack_term { tab; infos = old_info; sigma } TOP (subs_id 0) c)
+    ~share:true (** Not used by cbv *)
```

Comment:
This is not in my backyard, but IMHO it looks highly suspicious in terms of data structurs. If you are introducing a `lazy`-specific option there, you should decouple `cbv` from closures used in `lazy`. Would that introduce a lot of code?

Reply:
No, not really. I don't know why cbv is reusing the same data structure as the kernel, I guess it's an inheritance from the past. This can be addressed in a later patch though.